lib/src/bootloader.rs

```diff
@@ -12,15 +16,44 @@ pub(crate) fn install_via_bootupd(
     device: &Utf8Path,
     rootfs: &Utf8Path,
     configopts: &crate::install::InstallConfigOpts,
+    via_loopback: bool,
```

Comment:
I think it'd be a bit simpler for us to inspect the target block device here and special case loopback, instead of passing down this boolean.

(But, not opposed to the bool either, it's OK)